### DISCUSSION TITLE
Dragging a URDF file into Studio does nothing

### DISCUSSION THREAD

**Discussion by jhurliman** [foxglove/community#567] [Q&A]
**Description**

- Version: 1.18.0-dev
- OS: MacOS
- Data source (e.g. bag file, mcap file, rosbridge, ROS 1/2 native): No data source connected
- ROS distro (if applicable): N/A

**Steps To Reproduce**

1. Start a fresh copy of Studio (delete app settings if needed)
2. Drag and drop a URDF file on to Studio
3. Observe that nothing happens. No feedback about whether the file was loaded or not, or any visual change in the app.

**Expected Behavior**

A 3D rendering of the URDF should appear on screen.

**Actual Behavior**

No visual change. The Chrome dev console shows that URDF parsing was happening under the hood, but this didn't result in any panel being created that is capable of 3D rendering.

**Comment by defunctzombie** [comment]
Discussed in slack and we've removed the bug label for now. We will re-visit the behavior our app should take once we ship new3d.
